features/step_definitions/means_report_steps.rb

```diff
@@ -0,0 +1,102 @@
+Given("I have completed a non-passported application with bank statement uploads") do
```

Comment:
Should this explicitly say that it's an employed application too? e.g. 'non-passported employed application'